### PR TITLE
Create a Servlet to test if everyone can extract data from the database.

### PR DESCRIPTION
**Pull request by walde363**
Create a Servlet to test if everyone can extract data from the database.

### CODE REVIEW

**Comment by erikard** [comment]
After adding the project ID, and giving access to us as members, it works well.

**Review by erikard — COMMENTED** [review]
This works.

**Comment by rjkroege** [comment]
Why close it? Land the servlet. You might need it again?

**Comment by rjkroege** [comment]
And... maybe everybody will need this?

**Comment by walde363** [comment]
Yes, we will need it, don't know why was closed.

File: src/main/java/com/google/sps/servlets/test.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException
+  {
+       Connection conn; // Cloud SQL connection 
+      try
+      {
+          conn = DriverManager.getConnection("jdbc:mysql:///team5project?cloudSqlInstance=wramos-sps-summer21:us-central1:team5project&socketFactory=com.google.cloud.sql.mysql.SocketFactory&user=root&password=spsTeam5&useSSL=false"); 
+          ResultSet rs = conn.prepareStatement("SELECT * FROM QUESTIONS").executeQuery(); 
+          List<JSONObject> resList = new ArrayList<JSONObject>();
```
Comment (rjkroege) [inline src/main/java/com/google/sps/servlets/test.java:25]: There's some weird spacing here and elsewhere?

File: src/main/java/com/google/sps/servlets/test.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+{
+  
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException
+  {
+       Connection conn; // Cloud SQL connection 
+      try
+      {
+          conn = DriverManager.getConnection("jdbc:mysql:///team5project?cloudSqlInstance=wramos-sps-summer21:us-central1:team5project&socketFactory=com.google.cloud.sql.mysql.SocketFactory&user=root&password=spsTeam5&useSSL=false"); 
+          ResultSet rs = conn.prepareStatement("SELECT * FROM QUESTIONS").executeQuery();
```
Comment (rjkroege) [inline src/main/java/com/google/sps/servlets/test.java:24]: Your SQL code would be more maintainable going forward if you always enumerate the columns that you want and add a `limit` clause.

**Review by rjkroege — APPROVED** [review]
Can you format your code with some kind of java pretty-printer? I like [astyle](http://astyle.sourceforge.net) for Java code. Let's land this now but please fix the formatting later.

**Pull request merged**